### PR TITLE
Improve the `OS.is_debug_build()` documentation

### PR DESCRIPTION
**Pull request by Calinou**
See https://godotengine.org/qa/58519/check-if-game-is-exported.

### CODE REVIEW

File: doc/classes/OS.xml [hunk 1]
```diff
@@ -613,9 +613,10 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the build is a debug build.
+				Returns [code]true[/code] if the Godot binary used to run the project is a [i]debug[/i] export template.
```
Comment (akien-mga) [inline doc/classes/OS.xml]: That could also be the `tools=yes` build, no?

Comment (Calinou) [reply doc/classes/OS.xml]: Indeed, but that should be clear with the line below.

Comment (akien-mga) [reply doc/classes/OS.xml]: Ah, yes. I'd put both on the same sentence, I didn't expect several "Returns true".
```
Returns [code]true[/code] if the Godot binary used to run the project is a [i]debug[/i] export template or if running in the editor.
```

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**